content/sensu-go/5.18/guides/use-federation.md

```diff
@@ -170,6 +170,12 @@ Etcd replicators use the [etcd make-mirror utility][12] for one-way replication
 This allows you to centrally define RBAC policy on the `gateway` cluster and replicate those resources to other clusters in the federation, ensuring consistent permissions for Sensu users across multiple clusters via the `gateway` web UI.
 
 To get started, configure one etcd replicator per cluster for each of those RBAC policy types, across all namespaces, for each backend in the federation.
+
+{{% notice note %}}
+**NOTE**: Create a replicator for each resource you want to replicate within a namespace.
```

Comment:
```suggestion
**NOTE**: Create a replicator for each resource type you want to replicate within a namespace.
```

Reply:
Yes, I think you're right about that. fixing now...

Reply:
"within a namespace" confuses me here.